packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1299,6 +1316,16 @@ class Scaffold extends StatefulWidget {
   /// 20.0 will be added to `MediaQuery.of(context).padding.left`.
   final double drawerEdgeDragWidth;
 
+  /// This flag is deprecated and fixes and issue with incorrect clipping
+  /// and positioning of the [SnackBar] set to [SnackBarBehavior.floating].
+  @Deprecated(
+    'Fixes a bug that that fixes clipping and positioning of SnackBar '
```

Comment:
```suggestion
    'Fixes a bug that fixes clipping and positioning of SnackBar '
```

Reply:
I realized the wording in general was off, so I changed it to the following if that's okay:
```
    'Fixes a bug that incorrectly clips and positions SnackBar. It '
    'previously incorrectly offsets itself assuming a floating action button '
    'is present even when it is not. This parameter will be removed. '
    'This feature was deprecated after v1.15.3.'
```